### PR TITLE
August Design Update 

### PR DESCRIPTION
**Pull request by dcwalk**
## Features

- [x] Add in program and menu
- [x] Merge data files -> https://github.com/ournetworks/ournetworks.ca/pull/118/commits/71a14e5479fc481a429257bdce8bfdf7849b6ca3
- [x] Copy for registration on main page
  - [x] CTA in navigation
- [ ] Add in vector clip-art assets

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -19,11 +19,25 @@ This year will have a mix of scheduled talks and workshops alongside open space,
...
-Registration will be launching shortly!
+### Registration
 
-### Not Sure? Ask Us!
+We have three admission levels: Organization Supporter, Supporter, and Reduced Cost. While the cost for us to host this event is $78 per person, we want to ensure all can attend. We are looking for volunteers (in return for complimentary admission) and are able to offer a limited number of scholarships, please contact us at [{{ site.email }}](mailto:{{ site.email }})
 
-We have more open hours for people to ask questions and discuss activities for the open spaces at the conference. No prior registration needed, just visit: [https://meet.jit.si/ournetworks](https://meet.jit.si/ournetworks).
+| Admission              | Price* | Details |
+|:-----------------------|:------|:--------|
+| Organization Supporter | $110 | If your org or company is paying or your registration can be reimbursed, consider registering as organization supporter. This allows us to offer reduced cost admissions and scholarships. |
```
Comment (garrying) [inline index.md:28]:
```suggestion
| Organization Supporter | $110 | If your organization, or company is paying, or your registration can be reimbursed, consider registering as organization supporter. This allows us to offer reduced cost admissions and scholarships. |
```

File: index.md [hunk 2]
```diff
@@ -19,11 +19,25 @@ This year will have a mix of scheduled talks and workshops alongside open space,
...
+| Supporter              | $60 | The suggested admission for those employed or otherwise financially comfortable. At this rate we can sustain the event. |
+| Reduced Cost           | $25 | A lower cost admission for those who are unwaged, a student, or otherwise in need of a reduced cost. |
+
+<a class="button button-primary" href="https://www.brownpapertickets.com/event/4308694">{% include icons/ticket.svg %}&nbsp;&nbsp;Register on Brown Paper Tickets</a>
```
Comment (garrying) [inline index.md:32]:
```suggestion
<a class="button button-primary" target="_blank" href="https://www.brownpapertickets.com/event/4308694">{% include icons/ticket.svg %}&nbsp;&nbsp;Register on Brown Paper Tickets</a>
```

File: index.md [hunk 3]
```diff
@@ -19,11 +19,25 @@ This year will have a mix of scheduled talks and workshops alongside open space,
...
+
+<small><em>*Brown Paper Tickets charges CA$0.99 + 5.0% of the ticket price in buy fees.</em></small>
```
Comment (garrying) [inline index.md:34]:
```suggestion
<small><em>*Brown Paper Tickets charges CA$0.99 + 5.0% of the ticket price as a buyer fee.</em></small>
```

File: _layouts/splash.html [hunk 4]
```diff
@@ -9,7 +9,7 @@
   <hr>
   <div class="container">
   <h2 id="updates">Updates</h2>
-  <p>Want to receive an update when the conference program and registration launches?</p>
+  <p>Want to receive updates as we announce live stream details and more?</p>
```
Comment (garrying) [inline _layouts/splash.html:12]: Updated copy for live stream

File: _data/sessions.yml [hunk 5]
```diff
@@ -1,368 +1,713 @@
...
+    sessionType: "orga"
+    date: 2019-09-22
+    length: 0.5
+
+  - sessionID: 19003
+    title: "DHT, BFT, OMG: Know your building blocks for decentralized applications (2019 edition)"
+    descriptionLink: "https://github.com/ournetworks/2019-submissions/issues/3"
+    presenterID:
+      - kirill
+    description: >
```
Comment (garrying) [inline _data/sessions.yml:403]: @dcwalk Noticing that double-quotes are showing as the first character on session captions and it looks like this is causing it. Is there a case for descriptions to use `>`?

Comment (dcwalk) [reply _data/sessions.yml:403]: I think I mucked this up -- I was having issues with description length in just double quotes, so I switched to multiline, cleaned up in https://github.com/ournetworks/ournetworks.ca/pull/118/commits/5a7ee89bc466181a5d905e0eadee49d84cd4591f

**Comment by dcwalk** [comment]
Hey okay, I think we're there @garrying -- thank you review and all the extra bits :)))

**Pull request merged**